web/src/main/java/se/inera/intyg/webcert/web/service/facade/util/UtkastToCertificateConverterImpl.java

```diff
@@ -134,6 +143,15 @@ private Unit getCareProvider(Utkast certificate) {
             .build();
     }
 
+    private Unit getCareUnit(Utkast certificate) {
+        final var careUnitId = hsatkOrganizationService.getHealthCareUnit(certificate.getEnhetsId()).getHealthCareUnitHsaId();
+        final var careUnit = careUnitId != null ? hsatkOrganizationService.getUnit(careUnitId, "basic") : null;
```

Comment:
What does the magic "basic" do? Maybe a constant?

Reply:
Can a certificate be without a unit? Should it be considered an error and throw exception instead?